css/style.css

```diff
@@ -136,3 +153,91 @@ main {
   width: 14px;
   height: auto;
 }
+
+.rel-links {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 8px 0;
+  list-style-type: none;
+}
+
+.rel-links img {
+  width: 25px;
+  height: auto;
+}
+
+#alt-img {
+  height: 12px;
+}
+
+.card-section {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  gap: 25px;
+  margin-top: 85px;
+}
+
+.cards {
+  max-width: 320px;
+  height: min-content;
+}
+
+.cards img {
+  width: 320px;
+  height: auto;
+  object-fit: contain;
+}
+
+.card-txt {
+  color: #fff;
+  padding: 5px 15px;
+}
+
+.card-txt h3 {
+  margin: 30px 0;
+}
+
+.card-footer-listItem {
+  margin-right: 6px;
+}
+
+.card-footer-links {
+  text-transform: capitalize;
+  font-size: medium;
+}
+
+.card-footer,
+.card-footer-list {
+  display: flex;
+  justify-content: start;
+  flex-direction: column;
+  gap: 6px;
+  list-style: none;
+}
+
+.alt-menu {
+  flex-direction: row;
+  margin: 12px 0;
+}
+
+.grid-section {
+  display: grid;
+  grid-template-rows: repeat(1fr);
+  margin: 15px 0;
+}
```

Comment:
- [ ] Please use the proper background image with the shapes that is used in the Figma template, and then use the right background color for this section. You can use [this link](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A2346) to go directly to the background image with shapes used. 😁😁

Reply:
Changes effected here